### DISCUSSION TITLE
Trouble cloning a github repo for a school project

### DISCUSSION THREAD

**Discussion by SethMJohnston** [community/community#27986] [Other Feature Feedback, Questions, & Ideas]
Hi i am having trouble cloning a GitHub repo into my vs code using the command git clone <https copied link>
- the repo is private and part of a GitHub team repository where all 5 of us are admins
All of my teammates have successfully cloned the repo no problem but here I am stuck with this message when i try to clone: 
remote: Repository not found.
fatal: repository 'https repo link' not found
I have been scouring the internet as to why this is occurring but i am not sure what is going on anymore

**Comment by jfullstackdev** [comment]
try cloning it using the CMD, in order to see where the error is coming from 

if you have GitHub Desktop, you can try it too so you can pin down possibilities, for example, you might think you have an access to that repo but you don't have, say, you are using another account or maybe a misconfiguration in VS Code

**Comment by airtower-luna** [comment]
> remote: Repository not found.
> fatal: repository 'https repo link' not found

Assuming the repository link is correct, most likely your authentication isn't correct. You get the same error messages if a repository doesn't exist or you don't have access, to prevent people from discovering which repositories they don't have access to exist.

How are you doing authentication in VS Code? If you're asked for a password, you need to provide a [Personal Access Token (PAT)](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) instead. The token needs at least the **repo** scope, and also **workflow** if you want to use Actions.
